src/applications/vaos/tests/e2e/flows/community-care.cypress.spec.js

```diff
@@ -158,13 +154,14 @@ describe('VAOS community care flow using VAOS service', () => {
     );
     cy.axeCheckBestPractice();
     cy.findByText(/Choose a provider/).click();
-
-    cy.findByLabelText(/doe, jane/i).click();
-    cy.axeCheckBestPractice();
-    cy.findByText(/Choose provider/i).click();
-    cy.findByText(/remove/i).click();
-    cy.axeCheckBestPractice();
-    cy.findByText(/cancel/i).click();
+    cy.wait('@v1:get:provider', { timeout: 90000 }).then(() => {
+      cy.findByLabelText(/doe, jane/i, { timeout: 90000 }).click();
+      cy.axeCheckBestPractice();
+      cy.findByText(/Choose provider/i).click();
+      cy.findByText(/remove/i).click();
+      cy.axeCheckBestPractice();
+      cy.findByText(/cancel/i).click();
+    });
```

Comment:
Attempt to fix e2e stress test failure by waiting 90 seconds for the api call to finish.

Reply:
Do you know why api call time is taking longer?

Reply:
I wish I knew.